components/dropdown.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+
+import React, { FC, useState } from 'react';
+
+interface DropdownProps {
```

Comment:
You do not need to manually define your types. Flowbite has built-in types for these kinds of things. Please read the docs.
![image](https://github.com/Nidec-Control-Techniques-Dev/sales-advisor-pwa/assets/50437107/2e3d5f7a-fca9-47a9-aa75-d82291cd3d25)